### PR TITLE
Azure serverless sample application

### PR DESCRIPTION
**Pull request by aslihanilgin**
- Added Azure Function sample application to demonstrate MQ serverless applications on Azure
- Corrected typos in AWS Lambda serverless application README.md

### CODE REVIEW

**Review by chughts — CHANGES_REQUESTED** [review]

**Comment by aslihanilgin** [comment]
@chughts Requested changes made.

File: serverless/azure-functions-rest-http-trigger/.azure/.local_context_aslihanilginokan [hunk 1]
```diff
@@ -0,0 +1,9 @@
+; This file is used to store persistent parameters.
```
[the remark this answers is not in the document]
Comment (chughts) [reply serverless/azure-functions-rest-http-trigger/.azure/.local_context_aslihanilginokan:1]: This file is currently still part of the pull request.

File: serverless/azure-functions-rest-http-trigger/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
[the remark this answers is not in the document]
Comment (chughts) [reply serverless/azure-functions-rest-http-trigger/.vscode/settings.json:1]: These files are still part of the pull request.

File: serverless/azure-functions-rest-http-trigger/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# IBM MQ Serverless Sample for Azure Functions
+These REST samples are written in Node.js and have been tested with Azure Functions runtime of `Node.js`.
```
[the remark this answers is not in the document]
Comment (chughts) [reply serverless/azure-functions-rest-http-trigger/README.md]: A version of 16 contradicts the version 14 as specified in serverless/aws-lambda-rest/README.md

**Comment by chughts** [comment]
@aslihanilgin The hidden files need to be removed from the pull request. Updates to the .gitignore only apply to new files that you don't wish to commit, and not to files you have already committed.

**Comment by aslihanilgin** [comment]
Working on removing the sensitive and unneeded files

File: serverless/azure-functions-rest-http-trigger/.azure/.local_context_aslihanilginokan (hunk 1 above)
[the remark this answers is not in the document]
Comment (aslihanilgin) [reply serverless/azure-functions-rest-http-trigger/.azure/.local_context_aslihanilginokan]: This is now removed.

File: serverless/azure-functions-rest-http-trigger/.vscode/settings.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (aslihanilgin) [reply serverless/azure-functions-rest-http-trigger/.vscode/settings.json]: Now removed.

File: serverless/azure-functions-rest-http-trigger/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (chughts) [reply serverless/azure-functions-rest-http-trigger/README.md]: This conflict is still present. 

serverless/azure-functions-rest-http-trigger/README.md

> These REST samples are written in Node.js and have been tested with a Azure Functions runtime of `Node.js v16`.

serverless/aws-lambda-rest/README.md

> These REST samples are written in Node.js and have been tested with
>  a SAM AWS Lambda runtime of `nodejs14.x`

.

[the remark this answers is not in the document]
Comment (chughts) [reply serverless/azure-functions-rest-http-trigger/README.md]: AWS-lambda is not part of this pull request!

File: serverless/azure-functions-rest-http-trigger/lib/mqrestapi.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+// This class acts as the REST CAll manager. All MQ Rest calls are invoked
+// as methods on this class.
+class MQRestAPI {
+  constructor() {}
+
+  browseForMsg(options) {
+    return new Promise(function resolver(resolve, reject) {
+      //console.log('Configuration looks like ', config);
+      //console.log('Connection information looks like ', connection);
+      console.log('https://' + options.host + options.path);
```
[the remark this answers is not in the document]
Comment (chughts) [reply serverless/azure-functions-rest-http-trigger/lib/mqrestapi.js:38]: debug would be better than console.log

**Comment by chughts** [comment]
Please change the console.log logging to debug logging.

**Comment by aslihanilgin** [comment]
Changed console.log logging to debug logging.

**Pull request merged**